libs/ui-watt/src/lib/components/drawer/watt-drawer.component.spec.ts

```diff
@@ -14,44 +14,129 @@
+
+  // Fakes
+  const closedOutput = jest.fn();
+
+  // Setup
+  async function setup(story: Story<Partial<WattDrawerComponent>>) {
+    const { component, ngModule } = createMountableStoryComponent(
+      story(
+        { closed: closedOutput as unknown as EventEmitter<void> },
+        {} as never
+      )
+    );
+    await render(component, { imports: [ngModule] });
+  }
+
+  it('should open drawer', async () => {
+    await setup(Drawer);
+
+    expect(getDrawerContent()).not.toBeInTheDocument();
+
+    userEvent.click(getOpenDrawerButton());
+
+    expect(getDrawerContent()).toBeInTheDocument();
   });
 
-  /*
-  it('should open drawer', () => {
+  it('should not add content more than once, when "open" is called multiple times', async () => {
+    await setup(Drawer);
 
-  });
-
-  it('should not load content, before the drawer is opened', () => {
+    userEvent.click(getOpenDrawerButton());
+    userEvent.click(getOpenDrawerButton());
 
+    expect(getDrawerContent()).toBeInTheDocument();
   });
 
-  it('should not add content more than once, when "open" is called multiple times', () => {
+  it('should not load content, before the drawer is opened', async () => {
+    await setup(Drawer);
+
+    userEvent.click(getOpenDrawerButton());
 
+    await waitFor(() => {
+      expect(getInitialTimer()).toBeInTheDocument();
+    });
   });
 
-  it('should close drawer, triggered externally outside of the drawer', () => {
+  it('should close drawer, triggered externally outside of the drawer', async () => {
+    await setup(Drawer);
 
+    userEvent.click(getOpenDrawerButton());
+    userEvent.click(getExternalCloseDrawerButton());
+
+    expect(getDrawerContent()).not.toBeInTheDocument();
   });
 
-  it('should close drawer, triggered internally inside of the drawer', () => {
+  it('should close drawer, triggered internally inside of the drawer', async () => {
+    await setup(Drawer);
 
-  });
+    userEvent.click(getOpenDrawerButton());
+    userEvent.click(getInternalCloseDrawerButton());
 
-  it('should destroy content when closing', () => {
+    expect(getDrawerContent()).not.toBeInTheDocument();
+  });
 
+  it('should destroy content when closing', async () => {
+    jest.useFakeTimers();
+    await setup(Drawer);
+
+    userEvent.click(getOpenDrawerButton());
+    await waitFor(
+      () => {
+        expect(getStartedTimer()).toBeInTheDocument();
+      },
+      { timeout: 2000 }
+    );
+    userEvent.click(getInternalCloseDrawerButton());
+    userEvent.click(getOpenDrawerButton());
+
+    await waitFor(() => {
+      expect(getStartedTimer()).not.toBeInTheDocument();
+      expect(getInitialTimer()).toBeInTheDocument();
+    });
+    jest.useRealTimers();
   });
 
+  it('should output `closed` when drawer is closed', async () => {
+    await setup(Drawer);
+
+    userEvent.click(getOpenDrawerButton());
+    userEvent.click(getInternalCloseDrawerButton());
 
-  */
+    expect(closedOutput).toHaveBeenCalled();
+  });
```

Comment:
praise: nice tests 💪

suggestion: add one test that shows that `closedOutput` is also called when closing the drawer from outside.